src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -834,8 +837,10 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
       handleApprove()
       return
     }
+
+    mixpanel?.track(MixPanelEvent.LpDepositPreview, confirmedQuote!)
```

Comment:
only want this to occur on `!isSweepNeeded` or will be double tracked on sweep

Reply:
Oh very smart - 637889b4d01a65074327e22b963889009db0225d